Makefile

```diff
@@ -83,22 +83,16 @@ push-image:
 push-image-docker: CONTAINER_ENGINE=docker
 push-image-docker: push-container
 
-bonfire_process: $(BONFIRE_CONFIG)
+bonfire_process: 
 	@bonfire process -c $(BONFIRE_CONFIG) $(CLOWDAPP_NAME) \
 		-p service/IMAGE=$(IMAGE) -p service/IMAGE_TAG=$(IMAGE_TAG) -n $(NAMESPACE)
 
-bonfire_write_config: $(BONFIRE_CONFIG)
-$(BONFIRE_CONFIG):
-	@sed "s|##BONFIRE_REPODIR##|${PWD}|;\
-	s|##BONFIRE_APPNAME##|$(CLOWDAPP_NAME)|;\
-	s|##BONFIRE_CLOWDAPP_TEMPLATE##|$(CLOWDAPP_TEMPLATE)|" < templates/bonfire_config > $(BONFIRE_CONFIG)
```

Comment:
@psav  ping ? 

Also, if `templates` is no longer required, the one-level nesting with `src/backend-starter-app` is no longer needed for setup tools to figure out where are the files to build. 

Please, if you can confirm we can remove the `templates`, do so and move back to the root the app project's root dir ( move `src/backend-starter-app`  to ->  `backend-starter-app` and delete the `src` dir.

Reply:
Actually - I have second thoughts here. I thought there would be issues with the module path, but when packaged and installed from another Django project, there will be no issues, since the module path will start with `django-starter-app` or the name the user chose. 

probably disregard removing the `src` directory ... the `templates` dir removal comment stands though, specially if it's no longer required.

Reply:
I removed the old templates directory. It still runs, builds and deploys as expected.